src/Kiota.Builder/Configuration/GenerationConfiguration.cs

```diff
@@ -38,6 +38,7 @@ public ConsumerOperation? Operation
     public string ApiManifestPath { get; set; } = "apimanifest.json";
     public string OutputPath { get; set; } = "./output";
     public string ClientClassName { get; set; } = "ApiClient";
+    public string? ClientClassAccessModifier { get; set; }
```

Comment:
This could probably be an enum. of this type. https://github.com/microsoft/kiota/blob/main/src/Kiota.Builder/CodeDOM/AccessModifier.cs